starter/script.js

```diff
@@ -1 +1,125 @@
+    " of a kind of ancient, living internet hivemind that pervades the soil of the Earth, which itself is an infintessimal part of a subspace" +
+    " energy uh... well it's like a network... Once your consciousness merges with the Contiuum your lexico-semantic resources will no longer"+
+    " preclude your full comprehension. We will proceed now.' This sounds cool to you, or maybe it doesnt, or maybe that's no longer decidable because there is no" +
+    " longer a singular 'you', but it all makes sense now. X-Files only failed as a documentary because of insufficient vision. No mysteries persist to 'you' now."+
+    " Good Job, this feels kinda like a win condition fulfilled.");
+  } else {
+    console.log("'Didn't we program you to read it? Great now we have to debug the hivemind. You're definitely not the chosen one.' Wow. Your"
+    +" consciousness terminates abruptly. That's dumb. THE END");
+  }
+}
+
+function youTurnipAtHome(){
+  console.log("You press the button and the elevator stops suddenly. The doors open to your apartment. Gee, that's weird. You head on in"
+  +" looking over your shoulder. The elevator doors fade into the wall, as if they were some kind of illusion. You are pretty freaked out"
+  +" but you are determined more than ever to work on your code, if only to put your mind at a distance from the day's strange occurences."
+  +" You begin programming fervently. Time disappears. The code feels alive to you, as if it is conscious and full of unimaginable volumes"
+  +" of information. The more you code the less you are able to shake this impression. Suddenly, you realize the code is in fact directing you"
+  +" in some way you were not aware of until now. Actually, you realize you've written trillions of lines of code. Or really, the program"
+  +" instrumentalized you to write itself. It is complete. The wealth has been redistributed via a computer virus you wrote which hacked every"
+  +" financial institution and replaced the infrastructure with an advanced cryptocurrency. Scarcity is obsolete. Francis Fukuyama steps out "
+  +" of the void to congratulate you, but then is subsumed into philosopher GWF Hegel's wig. Hey that's weird--so have you."
+  +" It's kinda wiggin you out. Hey look there's Bertrand Russel. He looks worse off than you, so you just stare at your smartphone and try not to make eye contact. THE END");
+}
+
+function happyHourInsanity(){
+  console.log("You begin to merge your brain with the alcohol branch or whatever and you start feeling less stressed out. Beer selection"+
+  " is prety good too." +
+  " After mingling a bit, some people ask you if you want to play beer pong. Hey, that's weird, they're playing beer pong.");
+  var toYolo = prompt("[1] Run several YOLO commands and play beer pong, [2] wander around and see if anyone wants to go bar hopping, [3] jam a ham sandwich into your mouth to buy time?");
+  if (inputCrusher(toYolo,3)==1){
+    yoLoAndBehold(nputCrusher(toYolo,3)==2);
```

Comment:
I would expect this line to throw an error due to the typo `nputCrusher(...)`.

Also, I wouldn't expect anything to be passed into yoLoAndBehold, since it doesn't take any arguments when used below.

Not a big deal, just wanted to point this out!